packages/lib/rateLimit.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import LRU from "lru-cache";
```

Comment:
What's the benefit of using this over `memory-cache` ?

Reply:
I just ported the work from https://github.com/calcom/website/pull/70 

I can replace it for memory-cache =) good catch!

we can look into rate limiting requests later with lru cache or similar for other api endpoints, maybe even a rateLimiting middlware/handler

Reply:
Looking at google because I had no idea but this is the difference over both:
`**Least-recently-used (LRU) evicts the key-value used-the-least when the cache is full and it needs to add a value. Whereas a MemoryCache evicts the oldest key-values, or those past their 'use-by-date' if they happen to have one.Sep 19, 2015**`

Reply:
I'm just trying to keep dependencies as low as possible. Feel free to migrate either of those.

Reply:
Fixed and using memory-cache now!